.eslintrc

```diff
@@ -5,11 +5,7 @@
 		"plugin:@typescript-eslint/recommended",
 		"plugin:prettier/recommended"
 	],
-	"plugins": [
-		"react",
-		"@typescript-eslint",
-		"prettier"
-	],
+	"plugins": ["react", "@typescript-eslint", "prettier"],
```

Comment:
prettier actually [recommends](https://prettier.io/docs/en/integrating-with-linters.html) to just run the prettier command separately instead which I've been doing for a while now and works fine.

Reply:
True. Is done.